tests/test_grammar.py

```diff
@@ -1,304 +1,437 @@
 from __future__ import absolute_import
+import re
 
 import os
-from unittest import TestCase, main
+from unittest import TestCase, main, SkipTest
 
 from lark import Lark, Token, Tree, ParseError, UnexpectedInput
 from lark.load_grammar import GrammarError, GRAMMAR_ERRORS, find_grammar_errors, list_grammar_imports
 from lark.load_grammar import FromPackageLoader
+from lark.lark_validator import LarkValidator
+
+
+__all__ = []
+
+class LarkDotLark:
+    def __init__(self, grammar, **kwargs):
+        options = {}
+        options.update(kwargs)
+        if "start" in options and options["start"] != "start":
+            # We're not going to parse with the parser, so just override it.
+            options["start"] = "start"
+        options["propagate_positions"] = True
+        lark_parser = Lark.open_from_package("lark", "grammars/lark.lark", **options)
```

Comment:
Why are you passing `options` when you parse the `lark.lark` grammar? I think the right thing here is to ignore them, because they shouldn't affect parsing the grammar.

Reply:
`lark.lark` requires the starting rule to be named "start".  Some of the tests don't have one, or have another starting rule.  For example, `test_override_terminal()`.  Without overriding `options['start']`, `lark.lark` won't load those test grammars.

The `propagate_positions` is a debugging leftover.  I'll remove it.

Reply:
`propagate_positions` fixed by commit 0d01fe224fa05fd3575064247c3e8d749ea2e84f.